glpi/files/etc/php/conf.d/glpi.ini

```
@@ -2,3 +2,10 @@
 ; Session cookies security
 session.cookie_httponly = on
 session.cookie_samesite = "Strict"
+
+; opcache settings (similar to those of glpi private cloud)
```

Should we publicly document what is used in our private cloud ?

```suggestion
; opcache settings
```